scripts/falco-driver-loader

```diff
@@ -144,6 +148,14 @@ get_target_id() {
 		KERNEL_RELEASE="${VERSION_ID}"
 		TARGET_ID=$(echo "${OS_ID}" | tr '[:upper:]' '[:lower:]')
 		;;
+	("minikube")
+		TARGET_ID="${OS_ID}"
+		# Extract the minikube version. Ex. With minikube version equal to "v1.26.0-1655407986-14197" the extracted version
+		# will be "1.26.0"
+		[[ $(cat ${HOST_ROOT}/etc/VERSION) =~ ([0-9]+(\.[0-9]+){2}) ]]
```

Comment:
I'd add the if check here, even if it should never break.
But at least if it breaks for any reason, we have a log/exit driver loader.

Reply:
:+1: